piqe_ocp_lib/api/resources/ocp_nodes.py

```diff
@@ -80,18 +80,47 @@ def get_total_memory_in_bytes(self):
             logger.info("Total memory in bytes : %s", total_memory_in_bytes)
         return total_memory_in_bytes
 
-    def watch_all_nodes(self):
-        # We need to determine if a use case for this exists.
-        pass
-
-    def watch_a_node(self, node_name):
-        pass
-
-    def create_a_node(self, node_name):
-        pass
-
-    def delete_a_node(self, node_name):
-        pass
+    def is_node_ready(self, node_name, timeout=300):
```

Comment:
We'll need to introduce tests for this. We can capture this in a separate PR.

Reply:
Yes, generally speaking, not sure how to best test watch type methods. Let's discuss in our automation meeting.